docs/rule/no-positional-data-test-selectors.md

```diff
@@ -0,0 +1,27 @@
+# no-positional-data-test-selectors
+
+:white_check_mark: The `extends: 'recommended'` property in a configuration file enables this rule.
```

Comment:
I think we can't add to recommended (adding to recommended is a breaking change, we'll have to queue this up for a major bump), can you remove?

```suggestion
```